contributors/THIRD-PARTY-EXTENSION.md

```diff
@@ -0,0 +1,104 @@
+## Introduction
+
+Welcome to the guide for developing external extensions in the `create-eth` repository. This document outlines the workflow for creating and testing your extensions.
+
+## Workflow Overview
+
+### Phase 1: Initial Setup and Development
+
+1. **Clone the `create-eth` Repository:**
+
+   ```bash
+   git clone https://github.com/scaffold-eth/create-eth.git
+   cd create-eth
+   ```
+
+2. **Run the Build Script:**
+
+   ```bash
+   yarn build:dev
+   ```
+
+3. **Run the CLI to Create a New Instance:**
+
+   ```bash
+   yarn cli
+   ```
+
+   This command will create a **new instance** which is similar to running `npx create-eth@latest`
+
+   The name mentioned for "Your project name" question will be used as the **extension name**. For example, if you provide `eip`, the final extension name will be `eip`.
+
+4. **Develop the Extension:**
+   Add new files within the new instance directory. For example:
+
+   ```bash
+   # In new terminal window
+   cd eip
+   mkdir -p packages/nextjs/app/my-eip
+   touch packages/nextjs/app/my-eip/page.tsx
+   # add content to my-eip/page.tsx
```

Comment:
I'd remove this example.

In the 4. **Develop the Extension:**:

- CD into the instance
- Develop (explain a bit what you can do: adding files/folders, templates, dependencies... for more info see TEMPLATE.md)